stips/astro_image/astro_image.py

```diff
@@ -668,68 +580,49 @@ def make_epsf_array(self, psf_type = 'normal'):
             self._log("info", "PSF File {} to be put at {}".format(psf_name, psf_cache_dir))
             self._log("info", "PSF File is {}".format(psf_file))
             if os.path.exists(psf_file):
-                from webbpsf.utils import to_griddedpsfmodel
-                if (self.psf_commands is None or self.psf_commands == ''):
+                try:
+                    from webbpsf.utils import to_griddedpsfmodel
                     self.psf = to_griddedpsfmodel(psf_file)
                     have_psf = True
+                except Exception as e:
+                    self._log("error", "Creating psf from file {}  failed with {}".format(psf_file, e))
 
         if not have_psf:
-            base_state = self.celery_state
-            update_state = "<br /><span class='indented'>Generating PSF</span>"
-            self.celery_state = base_state + update_state
             ins = self.psf_constructor
-            if self.psf_commands != '':
-                for attribute,value in self.psf_commands.iteritems():
-                    setattr(ins,attribute,value)
             ins.filter = self.filter
             ins.detector = self.detector
             scale = self.scale[0]
-            # First limit -- PSF no larger than detector
-            ins_size = max(self.xsize, self.ysize) * self.oversample
-            # Second limit -- PSF no larger than half of max convolution area.
-            conv_size = self.convolve_size // (2*self.oversample)
-            # Third limit -- prevent aliasing
-            limit = 60.
-            if psf_type == 'bright': limit = 120.
-            elif psf_type == 'xbright': limit = 240.
-            safe_size = int(np.floor(limit * self.photplam / (2 * self.scale[0])))
-            if safe_size <= 0:
-                safe_size = max(self.xsize, self.ysize)
-            msg = "PSF choosing between {}, {} and {}"
-            self._log("info", msg.format(ins_size, conv_size, safe_size))
-            fov_pix = min(ins_size, conv_size, safe_size)
+            fov_pix = PSF_BOXSIZE
```

Comment:
The field of view is not the same as the PSF_BOXSIZE, this should be closer to 300 pixels.

Reply:
Fixed out of band, due to misunderstanding.